djstripe/models.py

```diff
@@ -1377,10 +1377,6 @@ class Event(StripeObject):
     event that just happened. Events are processed in detail by their respective
     models (charge events by the Charge model, etc).
 
-    Events are initially **UNTRUSTED**, as it is possible for any web entity to
-    post any data to our webhook url. Data posted may be valid Stripe information,
-    garbage, or even malicious. The 'valid' flag in this model monitors this.
-
```

Comment:
Why's this removed?

Reply:
I think @kavdev might be thinking we do webhook signature verification already. But it's still open at #536.

Reply:
Moving this to WebhookEventTrigger